### DISCUSSION TITLE
ModuleNotFoundError: No module named 'torch_geometric.logging'

### DISCUSSION THREAD

**Discussion by Rvlis** [pyg-team/pytorch_geometric#4988] [Q&A]
Help!
My PyG version is 2.0.4, but when I try to follow the [example code of GATConv](https://github.com/pyg-team/pytorch_geometric/blob/master/examples/gat.py), I face the error of **ModuleNotFoundError: No module named 'torch_geometric.logging'**. 
I try to search this module on PyG's documentation, but has no info about it. Is it has been deprecated? Could you help me, please.
Thank you!

**Comment by rusty1s** [comment]
It just got recently added to PyG. You can either checkout the examples for version 2.0.4, or follow [these instructions](https://github.com/pyg-team/pytorch_geometric#installation).
